pymeasure/instruments/tektronix/tek371A.py

```diff
@@ -0,0 +1,859 @@
+            **kwargs
+        )
+
+        self.write_delay = write_delay
+        self.query_delay = query_delay
+        self.last_write_timestamp = 0.0
+        self.srq_called = False
+
+    # ############################################
+    # # Tektronix Curve Tracer 371A SCPI commands.
+    # ############################################
+
+    id = Instrument.measurement(
+        "ID?",
+        """Return the identification of the instrument (string) """,
+        get_process=lambda r:
+        "".join(r)
+    )
+
+    help = Instrument.measurement(
+        "HELp?",
+        """Return the list of valid commands for the instrument.""",
+        get_process=lambda r:
+        ", ".join(r)
+    )
+
+    front_panel_settings = Instrument.measurement(
+        "SET?",
+        """Return the actual front-panel settings of the instrument.""",
+        get_process=lambda r:
+        ",".join(r)
+    )
+
+    #########################################################################
+    # # COLLECTOR SUPPLY COMMANDS
+    #########################################################################
+
+    cs_breakers = Instrument.measurement(
+        "CSOut?",
+        """Return the actual collector HIGH VOLTAGE and HIGH CURRENT breakers settings
+        of the instrument.""",
+        get_process=lambda r:
+        "".join(r)
+    )
+
+    cs_polarity = Instrument.control(
+        "CSPol?", "CSPol %s",
+        """Control the collector supply polarity and mode.""",
+        validator=strict_discrete_set,
+        values=COLLECTOR_SUPPLY_POLARITY_MODES
+    )
+
+    cs_peakpower = Instrument.control(
+        "PKPower?", "PKPower %s",
+        """Control the collector supply peak power watts settings.""",
+        validator=strict_discrete_set,
+        values=COLLECTOR_SUPPLY_PEAKPOWER_VALUES,
+        get_process=lambda r:
+        float("".join(r.replace(" ", "")).replace("PKPOWER", ""))
+    )
+
+    cs_collector_supply = Instrument.control(
+        "VCSpply?", "VCSpply %f",
+        """Control the collector supply output level (from 0.0% to 100.0% in 0.1% increments).""",
+        get_process=lambda r:
+        float("".join(r.replace(" ", "")).replace("VCSPPLY", ""))
+    )
+
+    ################################################################################
+    # # CRT READOUT TRANSFER COMMANDS
+    # # Group of commands for reading the vertical and horizontal cursor parameters.
+    ################################################################################
+
+    crt_readout_h = Instrument.measurement(
+        "REAdout? SCientific",
+        """Return the vertical and horizontal cursor parameters.""",
+        get_process=lambda r:
+        float(r[0].replace(" ", "").replace("READOUT", ""))
+    )
+
+    crt_readout_v = Instrument.measurement(
+        "REAdout? SCientific",
+        """Return the vertical and horizontal cursor parameters.""",
+        get_process=lambda r:
+        float(r[1])
+    )
```

Comment:
The docstrings do not match the content!

Reply:
Solved. Thanks